include/translated-post.php

```diff
@@ -265,4 +265,42 @@ public function get_untranslated( $type, $untranslated_in, $lang, $search = '' )
 
 		return $return;
 	}
+
+	/**
+	 * Changes the language of a post, and updates its translations.
+	 *
+	 * @since 3.1
+	 *
+	 * @param int          $post_id Id of the post to change the language of.
+	 * @param PLL_Language $lang New language to set to the post.
+	 * @param string       $post_type Post type of the post to change language of.
+	 *
+	 * @return WP_Error|true
+	 */
+	public function update_language( $post_id, $lang, $post_type ) {
```

Comment:
This method could be pulled up in the `PLL_Translatated_Object` class.

Hint: `PLL_Translated_Term` should have a similar 'inline edit' method.